hotrod/pom.xml

```diff
@@ -275,7 +207,7 @@
                             <skip>${module.skipCopyDependencies}</skip>
                             <includeScope>runtime</includeScope>
                             <excludeScope>test</excludeScope>
-                            <excludeGroupIds>com.github.ben-manes.caffeine,org.infinispan,org.infinispan.protostream,org.wildfly</excludeGroupIds>
+                            <excludeGroupIds>com.github.ben-manes.caffeine,org.infinispan.protostream,org.wildfly</excludeGroupIds>
```

Comment:
Seems removing the `org.wildfly` from here does the trick.

Reply:
Yeah, I missed the force push to update this before, all fixed now.